js/components/closed-caption-multi-audio-menu/tab.js

```diff
@@ -19,24 +22,36 @@ var Tab = React.createClass({
           speed={1}
           horizontal={false}
         >
-          <ul className="oo-cc-ma-menu__list">
+          <ul
+            className="oo-cc-ma-menu__list"
+            role={CONSTANTS.ARIA_ROLES.MENU}
+          >
             {this.props.itemsList.map(function(el, index) {
```

Comment:
Can we use a more descriptive name for `el`? It's not clear whether this is a DOM element or an audio track, etc.

Reply:
I changed it to 'item' and really it can be everything (and audio, and CC, and something also what we pass to the component as parameter for the list).